.github/ISSUE_TEMPLATE.md

```diff
@@ -11,14 +11,12 @@ For Bug report or feature requests, please fill out the relevant category below
 
 - Operating System:
   - <!-- OS and version (e.g. Windows 10, Ubuntu 16.04...) -->
-- Installation type:
-  - <!-- binaries or from source  -->
 - Version or commit hash:
-  - <!-- Output of git rev-parse HEAD, release version, or repos file  -->
+  - <!-- from source: output of `git -C navigation2 rev-parse HEAD
+         apt binaries: output of `dpkg-query --show "ros-$ROS_DISTRO-navigation2"`
+                              or `dpkg-query --show "ros-$ROS_DISTRO-nav2-*`" -->
 - DDS implementation:
-  - <!-- rmw_implementation used (e.g. Fast-RTPS, RTI Connext, etc -->
-- Client library (if applicable):
-  - <!-- e.g. rclcpp, rclpy, or N/A -->
+  - <!-- rmw_implementation used (e.g. Fast-RTPS, RTI Connext, etc.) -->
```

Comment:
Also add to get the versions here since we've had versions break things.

Reply:
Which version? How can you tell which version of FastRTPS? OpenSplice?

Reply:
Same with navigation2, look at the debians (or if compiled from scratch)

Reply:
I do not understand. What particular data are you requesting? How is the user supposed to know which package to get the version of? rmw_fastrtps_cpp? fastcdr? fastrtps? rmw_fastrtps_shared_cpp? Something from apt?

Reply:
Presumably, the one they are using, by looking at the debians installed via rosdep for their installed versions. If they list `rwm_implementation` as `example rtps` then they should provide the version of `example rtps` to help catch issues with outdated / new bugs

Reply:
I would rather just leave it out for now. It’s hard enough to figure out whether your system is using libopensplice67 or libopensplice69, what packages to look for will differ by which middleware they are using, and it can be asked in the issue conversation if it seems pertinent.

Reply:
OK